.github/workflows/ci.yml

```diff
@@ -128,3 +129,28 @@ jobs:
       #     make -j
       #     ./build/riscv64-nemu-interpreter -b -d $TEST_HOME/riscv64-spike-H-ext-so $TEST_HOME/kvm
       #     make clean-all
+  V-ext-tests:
+    runs-on: nemu
+    continue-on-error: false
+    name: nemu - V-extension
+    timeout-minutes: 60
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup env
+        run: |
+          echo "NEMU_HOME=$GITHUB_WORKSPACE" >> $GITHUB_ENV
+          echo "TEST_HOME=/nfs/home/share/ci-workloads/V-extension-tests" >> $GITHUB_ENV
+          git fetch
+          git checkout rv64v
```

Comment:
To test rv64v branch only, should use `branches: [ rv64v ]` to specify the condition of CI and this CI file is updated only on the rv64v branch. No need to merge into master.

What's the target of this PR? To test RVV on rv64v branch?

If so, you can modify this PR over the rv64v branch, and set the CI condition. Then, github will automatically run CI on the rv64v branch.

Reply:
Yes, we only want to do this test on rv64v for now, we will need to add it to master later